circle.yml

```diff
@@ -266,28 +310,16 @@ commands:
             mkdir -p /tmp/tests/clang-tidy
             if [ -f clang-tidy.log ]; then cp clang-tidy.log /tmp/tests/clang-tidy; fi
             mkdir -p /tmp/tests/render
-            if [ -f render-test/tests_index.html ]; then cp render-test/tests_index.html /tmp/tests/render; fi
-            if [ -f render-test/render-tests_index.html ]; then cp render-test/render-tests_index.html /tmp/tests/render; fi
-            if [ -f render-test/query-tests_index.html ]; then cp render-test/query-tests_index.html /tmp/tests/render; fi
+            if ls render-test/*.html 1> /dev/null 2>&1; then cp render-test/*.html /tmp/tests/render; fi
+            mkdir -p /tmp/tests/metrics
+            if ls metrics/*.html 1> /dev/null 2>&1; then cp metrics/*.html /tmp/tests/metrics; fi
```

Comment:
We don't have anything in metrics yet 

@zmiao for render tests we push everything to `render-test/metrics...`, right?

Reply:
For "normal" render-test, the output is under render-test folder where the manifest.json is located. But here the binary-size manifest in under metric/, so the output is under this folder.